spec/support/quota_helper.rb

```diff
@@ -190,6 +190,28 @@ def build_vmware_service_item
         @service_request = build_service_template_request("vmware_service_item", @user, :dialog => {"test" => "dialog"})
       end
 
+      def build_google_service_item
+        small_flavor = FactoryGirl.create(:flavor_google, :ems_id => @ems.id, :cloud_subnet_required => false,
+                                          :cpus => 1, :cpu_cores => 1, :memory => 1024)
+        google_requested_quota_values =
```

Comment:
@billfitzgerald0120 
can we make this into a separate function similar to
https://github.com/billfitzgerald0120/manageiq/blob/c84870dd34f4ba407c94d39600d2c64cb2475f8f/spec/support/quota_helper.rb#L111

Reply:
Created separate function for google_requested_quota_values as requested.  Please review